### PR TITLE
files for task 2

### PR DESCRIPTION
**Pull request by Inderpreetsingh11**

### CODE REVIEW

**Comment by nataliexn615** [comment]
Reviewed by Natalie

**Review by nataliexn615 — COMMENTED** [review]
Good Work

**Review by nataliexn615 — APPROVED** [review]

**Comment by henrywan1920** [comment]
Cool

File: Task_1.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        os.mkdir('./scraped_articles')
+    name = url.split("/")[-1]
+    print(name)
+    fname = f'scraped_articles/{name}.txt'
+
+    # Code here - write a file using with (2 lines)
+    with open(fname + '.txt', 'w') as f:
+        f.write(text)
+
+    # Code ends here
```
Comment (henrywan1920) [inline Task_1.py:65]: No clean function be used.

File: Task_1.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+# Code ends here
+
+# function to get the html source text of the medium article
+def get_page():
+    global url
+    # Code here - Ask the user to input "Enter url of a medium article: " and collect it in url
+    url = input('Enter url of a medium article: ')
+    # Code ends here
+
```
Comment (henrywan1920) [inline Task_1.py:19]: Cool.

**Review by henrywan1920 — COMMENTED** [review]
Good.

**Pull request merged**